week0002/phucnm/C++/week002.cpp

```diff
@@ -0,0 +1,56 @@
+#include <cstdio>
+#include <cstring>
+
+void spiralPrinting(int nRow, int nCol) {
+    int spiralArray[nRow][nCol];
```

Comment:
Thank you for your update. I'm almost ready to accept your solution and want to suggest a small change as the code is unsafe due to undefined length of the array. What happens if someone passes 0 or -1?

```
$ g++ -Wall -Werror -pedantic week0002.cpp
week0002.cpp:5:26: error: variable length arrays are a C99 feature
      [-Werror,-Wvla-extension]
    int spiralArray[nRow][nCol];
                         ^
week0002.cpp:5:20: error: variable length arrays are a C99 feature
      [-Werror,-Wvla-extension]
    int spiralArray[nRow][nCol];
                   ^
2 errors generated.
```